python/ray/serve/air_integrations.py

```diff
@@ -45,6 +45,22 @@ def _load_predictor_cls(
     return predictor_cls
 
 
+def _unpack_tensorarray_from_pandas(output_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
```suggestion
def _unpack_tensorarray_from_pandas(output_df: "pd.DataFrame") -> "pd.DataFrame":
```

Pandas is a soft dep.

Reply:
ha this might be it ! I was wondering where did i introduce hard dep that failed minimal install. Let me make the change now.